Cython/Debugger/libcython.py

```diff
@@ -324,7 +324,7 @@ def print_stackframe(self, frame, index, is_c=False):
         sys.stdout.write('\n')
 
         try:
-            sys.stdout.write('    ' + source_desc.get_source(lineno))
+            sys.stdout.write('    ' + source_desc.get_source(lineno) + '\n')
```

Comment:
I think we can start using f-strings here as well.
```suggestion
            sys.stdout.write(f'    {source_desc.get_source(lineno)}\n')
```